Userland/Libraries/LibJS/AST.cpp

```diff
@@ -2415,6 +2673,7 @@ void UpdateExpression::dump(int indent) const
     }
 }
 
+// 14.3.2.1 Runtime Semantics: Evaluation, https://tc39.es/ecma262/#sec-variable-statement-runtime-semantics-evaluation
 Completion VariableDeclaration::execute(Interpreter& interpreter, GlobalObject& global_object) const
```

Comment:
This function also does lexical declarations I believe:
https://tc39.es/ecma262/#sec-let-and-const-declarations-runtime-semantics-evaluation

Reply:
Yep, done!